jquery.balancetext.js

```diff
@@ -282,7 +282,11 @@
     $.fn.balanceText = function (skipResize) {
         var selector = this.selector;
 
-        if (!skipResize && balancedElements.indexOf(selector) === -1) {
+        if (!selector && this[0].className) {
+            selector = "." + this[0].className;
```

Comment:
`className` might contain multiple classes. I'm not sure if it's best to match each class, or the first class, or...? It's hard to guess which one was the intended target, which is why I think the option of passing in a `selector` makes a lot more sense for the long run.

If you want to match each class, something like `"." + this[0].className.replace(/ /g, " .")` should do the trick. Or `this[0].className.split(" ").map(function(class) { return "." + class; }).join(" ");`, which maybe reads a bit better.